contracts/unstoppable/FirstChallenge.spec

```diff
@@ -0,0 +1,84 @@
+using UnstoppableLender as lender;
```

Comment:
do we need lender?

Reply:
I wanted to show that the users are actually 2 regular users and not contracts